src/main/java/com/tabnine/userSettings/AppSettingsComponent.kt

```diff
@@ -61,13 +62,20 @@ class AppSettingsComponent {
             debounceTimeComponent.isEnabled = false
         }
 
-        panel = FormBuilder.createFormBuilder()
+        val panelBuilder = FormBuilder.createFormBuilder()
             .addLabeledComponent("Log File Path (requires restart): ", logFilePathComponent, 1, false)
             .addLabeledComponent("Log level (requires restart): ", logLevelComponent, 1, false)
-            .addLabeledComponent("Min delay between completions (requires restart): ", debounceTimeComponent, 1, false)
+
+        if (!isFixedDebounceConfigured()) {
```

Comment:
Mmm I'm a bit worried that there will be some confused users, like if someone already set this value and now it disappears from the ui or something like that.

I guess there's not much we can do, so we just need to remember it. Maybe let Maya know about this change

Reply:
you're right, but this is for a limited time (until we determine the default value) and for new users.
we have no more than 20 users right now that use this setting, so I think it's ok..